pkg/database/realm.go

```diff
@@ -356,6 +357,30 @@ func NewRealmWithDefaults(name string) *Realm {
 	}
 }
 
+// DefaultSMSTextTemplate returns database.DefaultSMSTextTemplate.
+// Convenance for unitizing in HTML templates.
```

Comment:
```suggestion
// Convenance for utllizing in HTML templates.
```

Reply:
(And everywhere else)

Reply:
google disagrees

https://www.google.com/search?q=utllizing&oq=utllizing&aqs=chrome.0.69i59j0i13i433j0i13l8.605j1j1&sourceid=chrome&ie=UTF-8